restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -391,6 +424,12 @@ public void patchDoc(
           @NotNull(message = "payload not provided")
           @NotBlank(message = "payload must not be empty")
           String payload,
+      @ApiParam(
+              value = "Include this to put a time-to-live on the document in the data store",
+              required = false)
+          @QueryParam("ttl")
+          @Min(value = 1, message = "TTL value must be a positive integer.")
+          Integer ttl,
```

Comment:
wait this is patch, you can not submit the TTL here? we need a different parameter like `ttl-auto=true`, right?